scripts/auth-open.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
@mellenburg this script is generating dynamic configuration for the driver. Instead of passing it a static text file, the driver will execute it and consume it's STDOUT as a configuration JSON or YAML.

That's why it's purpose is not entirely clear. It does not really do anything if executed, yet it's not a "configuration file". It's more like a "configuration script".

@orlandohohmeier perhaps if we put it in `test_util` and call it something like `gen_driver_config.py` ?